src/test/java/task40/helpes/TableSortSearchPage.java

```diff
@@ -0,0 +1,97 @@
+package task40.helpes;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.Select;
+
+import java.util.List;
+
+public class TableSortSearchPage {
+    private final String URL = "https://demo.seleniumeasy.com/table-sort-search-demo.html";
+    WebDriver driver;
+
+    public TableSortSearchPage(WebDriver driver)
+    {
+        this.driver = driver;
+        this.driver.get(URL);
+        this.driver.manage().window().fullscreen();
+    }
+
+    public Select getWebElementByName(String name)
+    {
+        return  new Select(driver.findElement(By.name(name)));
+    }
+    public WebElement getWebElementById(String id)
+    {
+        return driver.findElement(By.id(id));
+    }
+
+    public void collectTableDataList(String tableCssSelector, List<Employee> list)
+    {
+        WebElement table = driver.findElement(By.cssSelector(tableCssSelector));
+        List<WebElement> rows =
+                table.findElements(By.tagName("tr"));
```

Comment:
this one it better to update to find all 'tr' in 'tbody'. In such case you get rows without header row